.github/CONTRIBUTING.md

```diff
@@ -1,7 +1,15 @@
 # Contributing
 
-When contributing to this repository, please first discuss the change you wish to make via issue,
-email, or any other method with the owners of this repository before making a change.
+Thank you for your interest in contributing to our project! We welcome all contributions, whether it's a bug fix, feature suggestion, or documentation improvement.
```

Comment:
No need.

Reply:
I think keeping this, will be appreciate the aspiring contributors as it will help them feel more welcome to this project.